src/NavBrand.js

```diff
@@ -0,0 +1,74 @@
+import React, { cloneElement } from 'react';
+import BootstrapMixin from './BootstrapMixin';
+import classNames from 'classnames';
+
+import createChainedFunction from './utils/createChainedFunction';
+
+const NavBrand = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    navbar: React.PropTypes.bool,
```

Comment:
I suppose you meaning navbar attribute? I've seen that also other components like <code>NavItem</code> use it being default to false. I suppose that it is needed when you either use it as an independent component or inside a <code>Navbar</code> component.